### PR TITLE
Update topics-report.md

### PR DESCRIPTION
**Pull request by freetext-jyothi**

### CODE REVIEW

File: help/reports/topics-report.md [hunk 1]
```diff
@@ -1,83 +1,107 @@
...
+# **Topics Report**
+
+The Topics Table provides a breakdown of customer sentiment (positive, Negative, Neutral,Questions,Suggestions) categorized by various topics(specific things customers talk about in their feedback). This helps you understand what aspects of your product or service are most appealing to your customers and where there might be areas for improvement.
+
+
+## **Accessing the Topics Report**
 
 - Go to **Reports**.
 
 - Click on the **Topics Table**.
```
Comment (currents-dyuti) [inline help/reports/topics-report.md]: good to be as specific as possible e.g

- Click on **Topics Table** from the list of reports on the LHS panel.

**Review by currents-dyuti — APPROVED** [review]
run this through a chatgpt for basic spell check, types, cleaning.

File: help/reports/topics-report.md [hunk 2]
```diff
@@ -1,83 +1,107 @@
...
 
 Use **Filters** to select timeframes and analyze trends over specific periods like daily, weekly, monthly, or custom date ranges. Additionally, the filters provide various options for refining the data further.
 
-You can bookmark the report for easy access from the homepage and download it as CSV or PPT for further analysis.
+### **Drill Down into Topic Reviews**
 
 Clicking a topic or phrase within the Topics Table opens a window on the right-hand side (RHS) with all the reviews that mention the selected topic or contain the chosen phrase. Clicking a review opens a pop-up window displaying the complete review text and topics related to it. Additionally, the date the review was submitted is shown on the right-hand side (RHS) of the window. Add or remove topics by clicking on the edit icon.
 
-![](https://lh7-us.googleusercontent.com/oSYzbt54eYDIVG4C3xiAdbjxtmX9TcdP70sMsBABEmz0Kn6BrmFTCiZklgB2j-hkxURbA5ilGsS7yUNZRegxJi1EaI3Y4kzHI4xPNhj4vDhThxusGbw3oiIzwDDjV1cQcibydhkJGc5duDKpxbnK7vI)
+![](https://lh7-us.googleusercontent.com/yk6TyfbQrw7kwr52c7PHEN0iNfJM2SYtHyjgaIveH9YTzW1UzDaF_MTjJl2B1apadgWun_QPk5M2mUc7wxyH1XUGDC89LFiaDpZ2nfAR7iNSvVhSuoWVG-G98poZED3jcwYYT6-GjAqXQppoho9zW7Q)
```
Comment (currents-dyuti) [inline help/reports/topics-report.md]: you can change this to link to the locally uploaded image -

```
![](/img/help/reports/....)
```

**Review by freetext-jyothi — COMMENTED** [review]
Some of the images appear stretched and pixelated. Can I use this alt text code instead of alt text: path/to/image.png to resize the images , or is there another way?

**Pull request merged**